src/function-builder-detectors/zisi.js

```diff
@@ -64,6 +56,14 @@ module.exports = async function handler({ config, errorExit, functionsDirectory:
 
   const targetDirectory = await getTargetDirectory({ errorExit })
 
+  // Emptying the directory from which functions will be served, to clear any
+  // deleted functions from a previous run.
+  try {
+    await del(targetDirectory)
```

Comment:
Should `force: true` be used? (in case `process.cwd()` is not a parent of `targetDirectory`)

Reply:
Good point, thanks! Done in https://github.com/netlify/cli/pull/2197/commits/e50efb2d243459527f7b292d1f405571ef3f1ca7.